docs/detections/detection-engine-intro.asciidoc

```diff
@@ -73,7 +73,7 @@ Cold tier is a {ref}/data-tiers.html[data tier] that holds time series data that
 
 {es-sec} does *NOT* support cold tier data for the following {es} indices:
 
-* Index patterns controlled by {es-sec}, including signals and list indices
+* Index patterns controlled by {es-sec}, including alerts and list indices
```

Comment:
We've had some rendering issues with `es-sec` in the past, so this is just a small nit. 
```suggestion
* Index patterns controlled by {elastic-sec}, including alerts and list indices
```